extensions/iceberg/src/main/java/io/deephaven/iceberg/util/IcebergReadTable.java

```diff
@@ -0,0 +1,22 @@
+//
+// Copyright (c) 2016-2024 Deephaven Data Labs and Patent Pending
+//
+package io.deephaven.iceberg.util;
+
+import io.deephaven.annotations.BuildableStyle;
+import org.immutables.value.Value;
+
+/**
+ * Instructions for reading an Iceberg table as a Deephaven table.
+ */
+@Value.Immutable
+@BuildableStyle
+public abstract class IcebergReadTable extends IcebergReadOperationsBase {
```

Comment:
Would `IcebergReadOptions` be a better name for this? We have something named `Table` already and this confuses me.